sacrebleu/utils.py

```diff
@@ -305,7 +306,7 @@ def print_test_set(test_set, langpair, fields, origlang=None, subset=None):
 
     streams = [smart_open(file) for file in files]
     streams = filter_subset(streams, test_set, langpair, origlang, subset)
-    for lines in zip(*streams):
```

Comment:
Here, I'm not sure it's a good way to deal with "--echo" on "doc aligned" datasets.